app/uk/gov/hmrc/agentoverseasfrontend/views/application/anti_money_laundering.scala.html

```diff
@@ -18,75 +18,82 @@
 
-      <fieldset>
-
-        <legend>
-          <h1 class="heading-xlarge margin-bottom-20">@Messages("amls.title")</h1>
-        </legend>
-
-        <div class="panel panel-border-wide margin-bottom-30">@Messages("amls.inset.p1")</div>
+        <h1 class="govuk-heading-xl">@Messages("amls.title")</h1>
 
-        <div class="form-group @if(amlsForm("amlsBody").hasErrors){form-field--error soft--ends}">
-            <label for="amlsBody">@Messages("amls.form.supervisory_body")</label>
-            @amlsForm.error("amlsBody").map { formError => <span class="error-notification" id="amls-body-error">
-                <span class="visuallyhidden">@Messages("error.prefix")</span>
-                @Messages(formError.message)
-            </span>
-            }
-            <input
-            type="text"
-            class="form-control--block"
-            id="amlsBody"
-            name="amlsBody"
-            spellcheck="false"
-            @amlsForm.data.get("amlsBody").map { v => value="@v"}
-            @amlsForm.error("amlsBody").map { e => aria-describedby="amls-body-error"}
-            >
-        </div>
+        @govukInsetText(InsetText(
+            content = Text(Messages("amls.inset.p1"))
+        ))
 
-        <div class="form-group">
-        @uiInput(
-            amlsForm("membershipNumber"),
-            '_label -> Messages("amls.form.membership_number"),
-            '_inputClass -> "form-control--block",
-            '_labelClass -> "soft--ends",
-            '_labelTextClass -> "form-label"
-        )
-        </div>
+        @uiInput(Input(
+            id = "amlsBody",
+            name = "amlsBody",
+            inputType = "text",
+            classes = "form-control--block govuk-input--width-20",
+            spellcheck = Some(false),
+            label = Label(
+                classes = "form-label soft--ends",
+                content = Text(Messages("amls.form.supervisory_body")),
+            )
+        ).withFormField(amlsForm("amlsBody")))
 
-        <div class="form-group">
-            <details>
-                <summary>
-                    <span class="summary">@Messages(s"amls.hint.expandable")</span>
-                </summary>
-                <div class="panel panel-border-narrow" aria-hidden="true">
-                    <p>@Html(Messages(s"amls.hint.expandable.p1"))</p>
-                    <p>@Html(Messages(s"amls.hint.expandable.p2", "https://www.gov.uk/guidance/get-an-hmrc-agent-services-account"))</p>
-                </div>
-                <p>@Html(Messages(s"amls.hint.expandable.sign_out", routes.ApplicationSignOutController.signOut))</p>
-            </details>
-        </div>
+        @uiInput(Input(
+            id = "membershipNumber",
+            name = "membershipNumber",
+            classes = "form-control--block govuk-input--width-20",
+            label = Label(
+                classes = "form-label soft--ends",
```

Comment:
`form-label` is now `govuk-label` and soft--ends and form-control--block are old classes you can remove